SpatialGDK/Documentation/content/get-started/example-project/exampleproject-play.md

```diff
@@ -14,53 +14,61 @@ Find out more in the [glossary]({{urlRoot}}/content/glossary#launcher).
 
 ### Step 1: Set up a game client using the Launcher
 
-1. Open the SpatialOS Console at [console.improbable.io](https://console.improbable.io/) to see a list of your SpatialOS cloud projects. </br>
-The list will look something like this: </br>
+1. Open the SpatialOS Console at [console.improbable.io](https://console.improbable.io/) to see a list of your SpatialOS cloud projects. </br></br>
+The list looks something like this: </br></br>
 ![img]({{assetRoot}}assets/tutorial/console-projects-list.png)<br/>
 _Image: The Console home screen project list_
 </br></br>
-1. Select your SpatialOS cloud project name to see a list of deployments associated with that cloud project. </br>
-The list will look something like this: </br>
+1. Select your SpatialOS cloud project name to see a list of deployments associated with that cloud project. (In this example, the cloud project name is `gdc2019_demo`.)</br></br>
+The list looks something like this: </br></br>
 ![img]({{assetRoot}}assets/tutorial/console-deployments-list.png)<br/>
 _Image: The Console project screen deployment list_
 </br></br>
-1. Select your deployment name to see its overview screen.</br>
-The overview screen has a **Start** button  near the top. Select it to bring up the Start deployment dialogue box.
+1. Select your deployment name to see its overview screen.</br></br>
+The deployment overview screen looks something like this (but won't have a **Restart** button): </br></br>
+![img]({{assetRoot}}assets/tutorial/deployment-overview-screen.png)<br/>
+_Image: The Console deployment overview screen_
 </br></br>
-1. In the Start deployment dialogue box, select **Start**. 
-</br>
-Now the deployment is running, you see this near the top of the deloyment overview screen:
-![img]({{assetRoot}}assets/tutorial/console-launcher-button.png)<br/>
-_Image: The Console deployment overview screen showing the Launcher button_
-</br></br>
-1. In the deployment overview screeb, select the blue **Launch** button to open the Launcher dialog box.</br>
-It looks like this:</br>
-![img]({{assetRoot}}assets/tutorial/launch.png)<br/>
-_Image: The Console's Launcher dialog box_
-</br></br>
-1. Select **Launch** in the Launcher dialog box to download a game client.</br></br>
+
+1. Now start a deployment run. </br>
```

Comment:
I think we wanted to go for "launch" for deployments (according to the canonology), rather than "start" - but I do get that "start" makes sense here because of the button.

Reply:
Yeah, it's super, super confusing:
launch a deployment by spatial cloud launch
launch a deployment run by selecting start (or restart)
start the Launcher by selecting launch.
OMG!
Also the deployment dialog box says "Region" and the Console says "Cluster" for the dame thing.

Reply:
Have PM'ed. It's a mare.